source/agora/node/Network.d

```diff
@@ -0,0 +1,240 @@
+/*******************************************************************************
+
+    Contains the code used for network discovery.
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.node.Network;
+
+import agora.common.API;
+import agora.common.crypto.Key;
+import agora.common.Config;
+import agora.common.Data;
+import agora.common.Set;
+import agora.node.Client;
+
+import libsodium.crypto_sign_ed25519;
+import libsodium.randombytes;
+
+import vibe.d;
```

Comment:
Nope, this is fine in test code but not here

Reply:
You mean using the package module? Why is it not ok?

Reply:
Too many dependencies being pulled in

Reply:
Good point.